src/ui/georeferencer/qgsmapcoordsdialogbase.ui

```diff
@@ -16,67 +16,96 @@
-     </property>
-     <property name="wordWrap">
-      <bool>true</bool>
-     </property>
-    </widget>
-   </item>
-   <item row="1" column="2">
-    <widget class="QLabel" name="textLabel2">
-     <property name="text">
-      <string>Y / North</string>
-     </property>
-    </widget>
-   </item>
-   <item row="3" column="0">
-    <spacer name="verticalSpacer">
-     <property name="orientation">
-      <enum>Qt::Vertical</enum>
-     </property>
-     <property name="sizeHint" stdset="0">
-      <size>
-       <width>20</width>
-       <height>40</height>
-      </size>
-     </property>
-    </spacer>
-   </item>
-   <item row="1" column="3">
-    <widget class="QLineEdit" name="leYCoord"/>
-   </item>
-   <item row="1" column="1">
-    <widget class="QLineEdit" name="leXCoord"/>
-   </item>
-   <item row="4" column="0" colspan="4">
-    <widget class="QCheckBox" name="mMinimizeWindowCheckBox">
-     <property name="text">
-      <string>Automatically hide georeferencer window </string>
-     </property>
-    </widget>
+  <layout class="QGridLayout" name="gridLayout">
+   <item row="0" column="0">
+    <layout class="QVBoxLayout" name="verticalLayout">
+     <item>
+      <widget class="QLabel" name="label">
+       <property name="text">
+        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enter X and Y coordinates (DMS (&lt;span style=&quot; font-style:italic;&quot;&gt;dd mm ss.ss&lt;/span&gt;), DD (&lt;span style=&quot; font-style:italic;&quot;&gt;dd.dd&lt;/span&gt;) or projected coordinates (&lt;span style=&quot; font-style:italic;&quot;&gt;mmmm.mm&lt;/span&gt;)) which correspond with the selected point on the image. Alternatively, click the button with icon of a pencil and then click a corresponding point on map canvas of QGIS to fill in coordinates of that point.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+       </property>
+       <property name="wordWrap">
+        <bool>true</bool>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <widget class="QSplitter" name="splitter">
+       <property name="sizePolicy">
+        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
+         <horstretch>0</horstretch>
+         <verstretch>0</verstretch>
+        </sizepolicy>
+       </property>
+       <property name="orientation">
+        <enum>Qt::Horizontal</enum>
+       </property>
+       <widget class="QLabel" name="textLabel1">
+        <property name="text">
+         <string>X / East</string>
+        </property>
+       </widget>
+       <widget class="QLineEdit" name="leXCoord"/>
+       <widget class="QLabel" name="textLabel2">
+        <property name="text">
+         <string>Y / North</string>
+        </property>
+       </widget>
+       <widget class="QLineEdit" name="leYCoord"/>
+      </widget>
+     </item>
+     <item>
+      <widget class="QgsProjectionSelectionWidget" name="mProjSelect" native="true">
```

Comment:
Why having a projection selection widget here? We should deal with only two CRSes in the georeferencer context: the CRS set for the raster being georeferenced, and the main window's map canvas.

Reply:
I'd have to disagree, if you have a map with coordinates drawn on it and you need to type them manually, having the option to set the crs of the coordinate source is critical. This is why I have added this widget, though it can be wonky at time.

Reply:
I'd split your effort into two then. First, come up with a clean PR to do map canvas to georeferencing raster's projection transformation, that's a gap that's undeniably worth fixing.

In a second PR, work on polishing the code and UI to propose this additional feature.

Reply:
Welp, it works fine now, I just had to not declare the projection selector in the header. All now works as intended, projections are properly assigned and coords are converted. I'll be making a gif to display.

Reply:
@nirvn I'd like to keep this in if possible, it works far better this way, see the gif below to see it in action.